### PR TITLE
Missing functions

### PR DESCRIPTION
**Pull request by Muriel-Salvan**

### CODE REVIEW

File: src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala
Comment (ebastien) [inline src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala]: The signature of the function should be restricted to only match what is actually supported by Impala: 

https://www.cloudera.com/documentation/enterprise/5-5-x/topics/impala_math_functions.html#math_functions

> round(double a), round(double a, int d), round(decimal a, int_type d)

As a first iteration we could limit ourselves to `round(double a)`.

File: src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala
Comment (ebastien) [inline src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala]: Comment does not match with the name of the file.

File: src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala
Comment (ebastien) [inline src/main/scala/com/amadeus/ti/calcite/SqlRoundFunction.scala]: Not sure the ASF license applies here.

**Review by ebastien — CHANGES_REQUESTED** [review]

**Comment by Muriel-Salvan** [comment]
PR corrected per review comments, and also added support for SUBSTRING and CONCAT.
Waiting for further review.

**Review by ebastien — APPROVED** [review]

**Comment by Muriel-Salvan** [comment]
Can you merge it or give me write access to the repo?

**Pull request merged**
